napari/view_layers.py

```diff
@@ -109,7 +109,7 @@ def _merge_layer_viewer_sigs_docs(func):
 
 
 _viewer_params = inspect.signature(Viewer).parameters
-_dims_params = inspect.signature(Dims).parameters
+_dims_params = Dims.__fields__
```

Comment:
I do not like this but:
```ipython
In [6]: inspect.signature(Dims)
Out[6]: <Signature (**kwargs)>
```

Reply:
works for me

Reply:
I would like to know what happens that is examples are executed then the signature is properly recognized.